### ISSUE TITLE
[issue]: VentoyPlugson does not create the correct file name for its json file if the ISO partition is formatted as Ext4, causing the json file to not be read

### ISSUE THREAD

**Issue by NintendoManiac64** [ventoy/Ventoy#1898]
### Official FAQ

- [X] I have checked the official FAQ.

### Ventoy Version

1.0.80

### What about latest release

Yes. I have tried the latest release, but the bug still exist.

### Try alternative boot mode

Yes. I have tried them, but the bug still exist.

### BIOS Mode

Both

### Partition Style

MBR

### Disk Capacity

8GB, 128GB

### Disk Manufacturer

Unirex (8GB), Samsung (128GB; SATA SSD)

### Image file checksum (if applicable)

Yes.

### Image file download link (if applicable)

I just used the normal Linux download for Ventoy (and yes, I checked the SHA256)

### What happened?

Simply put, if your Ventoy ISO partition is formatted as Ext4, then VentoyPlugson will end up not creating the correct file name for its json file; this obviously causes the resulting json file to not be read.

**Comment by ventoy** [comment]
Please upload `VentoyPlugson.log` here for analyze.

**Comment by NintendoManiac64** [comment]
The log is pretty bare and doesn't look to indicate much to me, but I'll upload it anyway:

[VentoyPlugson.log](https://github.com/ventoy/Ventoy/files/9720568/VentoyPlugson.log)

You really might just have to test it out yourself - I tried it with two very different drives and both behaved the same way. You don't even have to do anything fancy, e.g. it can be a completely empty drive with no ISOs and, for VentoyPlugson, you similarly only have to change a single setting from the defaults.

**Comment by ventoy** [comment]
Please test as in the above photo and upload your result.

**Comment by NintendoManiac64** [comment]
Well then the next thing is maybe the OS and/or browser makes a difference?  For the 8GB USB drive I was using Linux Mint 20.3 Xfce (installed) paired with the Pale Moon browser while, for the 128GB SSD, I was using Linux Mint 21 Cinnamon (live ISO) paired with its pre-installed Firefox browser.

By the way, the Mint 21 and 128GB drive were also tested on a Thinkpad T420, so that's another variable we can rule out.

**Comment by steve6375** [comment]
Maybe you just pulled out the USB drive without doing a sync or eject first?

**Comment by NintendoManiac64** [comment]
Well this is interesting - when I used the exact same mounting options as you, it at least made it half of the way there.  Previously, on the 8GB + Mint 20.3 scenario, it kept creating a file named "ven" while, when I used your exact mounting options, it at least creates a file named "ventoy" but still with no file extension.

These were the default mounting options:
```
nosuid,nodev,nofail,noauto,x-gvfs-show
```

Also it can't be due to removing the USB drive because I would check the "ventoy" directory for the supposed json file before unmounting the drive.

That does remind me though - for the testing in VentoyPlugson, what I go is simply change any of the settings in "Global Configuration", most easily the win11 check since it's right at the top of the page.
